app/infrastructure/persistence/repositories/tenant_integrity_profile_history_repo.py

```
@@ -0,0 +1,75 @@
+        self,
+        *,
+        tenant_id: str,
+        previous_profile: str | None,
+        new_profile: str,
+        changed_by_user_id: str,
+        change_reason: str | None,
+        effective_from_seq: int,
+        cooling_off_ends_at: datetime | None = None,
+    ) -> TenantIntegrityProfileHistory:
+        """Insert a new history row and return it."""
+        now = datetime.now(timezone.utc)
```

Ruff (0.15.5)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)